### ISSUE TITLE
[Bug]: Unable to interact with Supplementaries Red Merchant

### ISSUE THREAD

**Issue by trevorjd** [AllTheMods/ATM-10#1539]
### Possible Fixes

No

### Modpack Version

2.7

### What happened?

The Red Merchant from Supplementaries mod can't be interacted with using a right-click.  Verified nothing in off-hand; even removed all armour and curios.  Nothing on server console nor client log when I try.

Supplementaries github has no reference to issues with that mob.  ATM Discord has plenty of reports of the same inability to interact and a few people replying that "it worked for me". 

I don't have the time today but I'll try testing Supplementaries in isolation tomorrow.

**Comment by upd0g1** [comment]
issue persists in atm10 v2.30 (current according to curseforge, prism launcher, and discord as of 1/30/2025)

**Comment by eSonOfAnder** [comment]
Have reported this to the Supplementaries GitHub, but they are unable to recreate the problem in isolation, so it appears to be something either pack config or mod interaction related: https://github.com/MehVahdJukaar/Supplementaries/issues/1332

**Comment by xtended2l** [comment]
Yep, I can confirm, this guy was never interact-able in modpack.

**Comment by TheBedrockMaster** [comment]
Is this still a problem? If so, it appears to be caused by Moonlight Lib.
